Package.swift

```diff
@@ -124,14 +124,21 @@ let package = Package(
         "DeclarationAttribute.swift.gyb",
       ]
     ),
+    .target(
+      name: "SwiftParserDiagnostics",
+      dependencies: ["SwiftBasicFormat", "SwiftDiagnostics", "SwiftSyntax"],
+      exclude: [
+        "CMakeLists.txt",
+      ]
+    ),
     .target(
       name: "SwiftOperators",
       dependencies: ["SwiftSyntax", "SwiftParser", "SwiftDiagnostics"]
     ),
     .target(
       name: "SwiftCompilerSupport",
-      dependencies: [
-        "SwiftSyntax", "SwiftParser", "SwiftDiagnostics", "SwiftOperators"],
+      dependencies: ["SwiftSyntax", "SwiftParser", "SwiftParserDiagnostics",
```

Comment:
You might need a paired PR to the Swift repository to link against `SwiftParserDiagnostics`.